packages/lib/client-shared/src/util.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Settings for calling {@link gpcArtifactDownloadURL} as parsed from an
+ * environment variable by {@link parseGPCArtifactsConfig}.
+ */
+export type GPCArtifactsConfigEnv = {
+  source: string | undefined;
+  stability: string | undefined;
+  version: string | undefined;
+};
+
+/**
+ * The version of the latest supported version of artifacts published on NPM.
+ */
+export const GPC_ARTIFACTS_NPM_VERSION = "0.0.2";
+
+/**
+ * Parse configuration overrides for GPC artifact download from a string
+ * environment variable, or provides default values if not set.
+ *
+ * @param envConfig environment variable value for override
+ * @returns config variables suitable for calling {@link gpcArtifactDownloadURL}.
+ */
+export function parseGPCArtifactsConfig(
+  envConfig: string | undefined
+): GPCArtifactsConfigEnv {
+  const defaultConfig = {
+    source: "unpkg",
+    stability: "prod",
+    version: GPC_ARTIFACTS_NPM_VERSION
+  };
+  if (
+    envConfig === undefined ||
+    envConfig === "" ||
+    envConfig === "undefined"
+  ) {
+    return defaultConfig;
+  }
+  const config = JSON.parse(envConfig);
```

Comment:
this `JSON.parse` could throw or be something other than an object - we should be robust to this and `console.warn` in those cases. potentially may be worth using `zod` so as to not reimplement safely parsing stuff

Reply:
I can put in an error log, but I feel like putting too much work into safe parsing of env variables is unnecessary.  So long as the dev who's setting up the server can see the error and fix it, I think it's fine to crash.  (That's distinct from how I'd feel about a similar case parsing input from the network.)